packages/gateway/src/features/transformation/index.ts

```diff
@@ -46,7 +47,7 @@ async function transformField(
     throw new Error(
       `Expected ${JSON.stringify(
         fieldBuilderForVal
-      )} to be a FieldBuilder object. The current field value is ${JSON.stringify(
+      )} to be a FieldBuilder object for field name ${currentPropName}. The current field value is ${JSON.stringify(
```

Comment:
I added these here as I was getting an error registering a declaration and the error message didn't have information about which field caused the error, making it a bit difficult to debug